### PR TITLE
Internal documentation, typos, remove misleading message

### PR DESCRIPTION
**Pull request by penrods**
* Made MycroftSkill.remove_event() return a bool, preventing unnecessary/misleading message from being posted by MycroftSkill.cancel_scheduled_event()
* More doc and several minor renames around intent processing
* Several minor typo and doc corrections

### CODE REVIEW

File: mycroft/skills/intent_service.py [hunk 1]
```diff
@@ -315,15 +332,16 @@ def parse_converse(self, utterances, lang):
                 return True
         return False
 
-    def parse_utterances(self, utterances, lang):
+    def _adapt_intent_match(self, utterances, lang):
         """
-            Parse the utteracne using adapt  to find a matching intent.
+            Run the Adapt engine to search for an matching the utterance
```
Comment (forslund) [inline mycroft/skills/intent_service.py]: `intent` or `intent handler` missing here (after *an*)

File: mycroft/client/enclosure/display_manager.py [hunk 2]
```diff
@@ -22,6 +22,23 @@
...
+# Currently, a wakeword sets the ActiveSkill to "wakeword", which will auto
+# clear after 10 seconds.
+#
+# A skill is set to Active when it matches an intent, outputs audio, or
+# changes the display via the EnclosureAPI()
+#
+# A skill is automatically cleared from Active two seconds after audio
+# output is spoken, or 2 seconds after resetting the disply.
+#
+# So it is common to have "" as the active skill.
```
Comment (forslund) [inline mycroft/client/enclosure/display_manager.py]: Feels like this should be a docstring instead of a comment block.

Comment (penrods) [reply mycroft/client/enclosure/display_manager.py]: Yeah, this was just notes to myself originally.  I made into a docstring.

File: mycroft/skills/intent_service.py (hunk 1 above)
Comment (penrods) [reply mycroft/skills/intent_service.py]: Cleaned up, along with more doc.

**Comment by forslund** [comment]
Looks good, merging.

**Pull request merged**